inject-generator/src/main/java/io/avaje/inject/generator/InjectProcessor.java

```diff
@@ -332,4 +334,11 @@ private void readInjectModule(RoundEnvironment roundEnv) {
         }
       });
   }
+
+  private void registerSPI(Set<? extends Element> beans) {
+    ElementFilter.typesIn(beans).stream()
+        .map(TypeElement::getQualifiedName)
```

Comment:
It looks like this needs to filter on types that implement InjectExtension?

Reply:
I have added a filter such that InjectProcessor only adds services that are InjectExtension ones.  Other services are expected to be handled by avaje-spi-service (or something else like avaje-jsonb or avaje-validator).